modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/package.scala

```diff
@@ -15,6 +15,7 @@ package engine {
   object HandleP {
     def fromProcess(p: Process[Task, Event]): HandleP[Unit] = HandleP(Applicative[Handle].pure[(Unit, Option[Process[Task, Event]])](((), Some(p))))
   }
+  case class ActionMetadata(conditions: Conditions, operator: Option[Operator], observer: Option[Observer])
```

Comment:
Nothing wrong with this but now we have 2 places in the engine where the conditions, the operator, and the observer are defined and would have to be kept in sync somehow.

Should we remove the other places where those parameters are defined and when needed globally obtain it on-the-fly from the underlying Actions? How do we handle a change in any of those parameters when updated from the UI?

For the record, when I worked on making the operator global for the engine I was against it but had to made it nevertheless, so I prefer the metadata as it is in this PR.